book/open_source.md

```diff
@@ -0,0 +1,54 @@
+# Open-source and Open Science Resources
+
+## What is Open Science?
+
+![open-science-defn](../img/open-science-defn.png)
```

Comment:
@JessicaS11 see the preview link above to ensure things look like you want them too. I think this needs to be `./img/` rather than `../img`

Reply:
thanks for catching that, @scottyhq!